core/jobs/src/test/java/com/cognifide/aet/job/common/comparators/source/SourceComparatorRegExpTest.java

```diff
@@ -26,12 +26,12 @@
 @RunWith(Theories.class)
 public class SourceComparatorRegExpTest {
```

Comment:
Could you please rename this class to `CodeFormatterTest`? As it contains unit tests for `CodeFormatter` now...

Reply:
Done